packages/xstate-solid/src/useMachine.ts

```diff
@@ -1,6 +1,6 @@
 import type { AnyStateMachine, StateFrom, InterpreterFrom, Prop } from 'xstate';
```

Comment:
likely this file should be removed (or rather renamed to `useActor`)

Reply:
I did and worked on making this more generic, but there are some type issues I need help with